integreat_cms/cms/views/mixins.py

```diff
@@ -92,15 +92,17 @@ def get_context_data(self, **kwargs):
         :return: The template context
         :rtype: dict
         """
+
         context = super().get_context_data(**kwargs)
+
+        _kwargs = {"region_slug": kwargs["region_slug"]}
+        if kwargs.get("language_slug") is not None:
+            _kwargs["region_slug"] = kwargs["language_slug"]
         context.update(
             {
                 "back_url": reverse(
                     self.back_url_name,
-                    kwargs={
-                        "region_slug": kwargs["region_slug"],
-                        "language_slug": kwargs["language_slug"],
-                    },
+                    kwargs=_kwargs,
```

Comment:
```suggestion
                    kwargs={
                        "region_slug": kwargs["region_slug"],
                        "language_slug": kwargs["language_slug"],
                    },
```
Since organizations aren't edited regularly and there is no large text field where simultaneous editing is a problem, we don't need the content edit lock for them.

Reply:
Your right! Thanks!